src/h5web/explorer/Explorer.tsx

```diff
@@ -1,76 +1,55 @@
-import { useEffect, useContext, ReactElement } from 'react';
-import { FiFileText } from 'react-icons/fi';
-import type { Entity } from '../providers/models';
+import { useEffect, useContext, ReactElement, useState, Suspense } from 'react';
+import { FiFileText, FiRefreshCw } from 'react-icons/fi';
 import EntityList from './EntityList';
 import styles from './Explorer.module.css';
-import { getEntityAtPath, getParents } from '../utils';
 import { ProviderContext } from '../providers/context';
-import { isGroup } from '../guards';
-import { useSet } from 'react-use';
 
 const DEFAULT_PATH = process.env.REACT_APP_DEFAULT_PATH || '/';
 
 interface Props {
-  onSelect: (entity: Entity) => void;
-  selectedEntity?: Entity;
+  onSelect: (path: string) => void;
 }
 
 function Explorer(props: Props): ReactElement {
-  const { onSelect, selectedEntity } = props;
-  const { domain, metadata: root } = useContext(ProviderContext);
+  const { onSelect } = props;
 
-  const [
-    expandedGroups,
-    { add: expandGroup, toggle: toggleGroup },
-  ] = useSet<string>();
+  const { domain } = useContext(ProviderContext);
+  const [selectedPath, setSelectedPath] = useState<string>(DEFAULT_PATH);
```

Comment:
The new local state of the `Explorer`. As explained, the selection of the `DEFAULT_PATH` is now trivial.